frontend/.eslintrc.json

```diff
@@ -26,30 +25,14 @@
         "components": ["a"]
       }
     ],
-    "indent": [
```

Comment:
as per the new "eslint-check" (which is part of the online documentation of one of those eslint-prettier-config-foo modules) these rules conflict with the prettier config